boot/boot.go

```diff
@@ -40,6 +42,40 @@ const (
 	TryingStatus = "trying"
 )
 
+var (
+	// EarlyRunMnt is the rootdir where ubuntu partitions are mounted during the
+	// early boot (i.e. the initramfs)
+	EarlyRunMnt = filepath.Join(dirs.GlobalRootDir, "run/mnt")
```

Comment:
are they all missing a Dir suffix ?

Reply:
s/early boot/early user space/   which makes me wonder if boot.Early* alone is confusing or not as way to call these

Reply:
yes, I will add Dir suffix to all of them and also change Early -> Initramfs everywhere